include/rocksdb/db.h

```diff
@@ -611,6 +611,12 @@ class DB {
     return GetIntProperty(DefaultColumnFamily(), property, value);
   }
 
+  // Reset internal stats for DB and all column families.
+  // Note this doesn't reset options.statistics.
+  virtual Status ResetInternalStats() {
```

Comment:
What are your thoughts about renaming this public API to ResetDBStats (or perhaps just ResetStats)? 
My arguments is that nothing 'internal' should be exposed in the public API, even in the name. If we name it as ResetDBStats we don't need to rename it (or add another API) later again if we ever decide to reset option.statistics as well; It would then just be an implementation change.